admin/views/tool-import-export.php

```diff
@@ -63,6 +63,16 @@
 
 }
 
+/**
+ * Filter: 'wpseo_import_tabs' - Allow adding custom import tabs to the page.
+ *
+ * Tabs should be added by setting a key. Value has to be an array with the keys tabvalue (the value display on the tab)
+ * and the tabcontent (the contents that will be shown when pressing on the tab)
+ *
+ * @api array - The array that will be use to fill the tabs.
```

Comment:
'use' -> 'used'

Please document the array structure you expect. Please include example values for the array keys you expect.